### PR TITLE
New availability page put together

### PR DESCRIPTION
**Pull request by YuraRiabov**
Screenshot_493

### CODE REVIEW

File: frontend/src/app/modules/availability/new-slot/new-availability/new-availability.component.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    }
+
+    private initializeSideMenu() {
+        this.sideMenuGroups = [{
+            items: [
+                { text: 'General', routerLink: 'general' },
+                { text: 'Booking page', routerLink: 'event-details' },
+            ],
+        }];
+    }
```
Comment (tatianahutii) [inline frontend/src/app/modules/availability/new-slot/new-availability/new-availability.component.ts]: Please move this to a separate file (helper)

**Review by tatianahutii — CHANGES_REQUESTED** [review]

**Comment by tatianahutii** [comment]
Could you also change header color to be more similar to what we have in our prototype?

File: frontend/src/app/modules/availability/new-slot/new-availability/new-availability.component.ts (hunk 1 above)
Comment (YuraRiabov) [reply frontend/src/app/modules/availability/new-slot/new-availability/new-availability.component.ts]: What exactly should it do, sorry, I haven't fully understood?

**Review by tatianahutii — APPROVED** [review]
LGTM

**Pull request merged**